.circleci/config.yml

```diff
@@ -58,21 +67,31 @@ jobs:
 
 workflows:
   version: 2
+  linters:
```

Comment:
can you also make it 3 different checks?

Reply:
I can make them into 4 different checks, but I'd rather not to keep the overview. With the current change there are clear purposes to the 4 workflows; testing, linting, building, and building + pushing. This is also in line with the changes I did for statutory.